src/components/BackendExportEntitiesForm.tsx

```diff
@@ -0,0 +1,257 @@
+              >
+                Save
+              </Button>
+              &nbsp;
+            </span>
+          ) }
+          { configurationSaving && <Progress className={classes.progress}/> }
+        </AccordionDetails>
+      </Accordion>
+      { (runCount === null || runCount > 0) && [
+        <div className={classes.executionNavHeader} key="nav-header" data-testid="execution-header">
+          { currentRunIndex > 0 && <Link data-testid="execution-header-prev" href="#" onClick={prev} className={classes.paginationLink}>&lt;&lt;&lt;</Link> }
+          { currentRunIndex <= 0 && <span data-testid="execution-header-prev">&lt;&lt;&lt;</span>}
+          &nbsp;
+          <div className={classes.executionNavHeaderText}>Showing execution {currentRunIndex + 1} of {runCount}</div>
+          &nbsp;
+          { currentRunIndex < (runCount || 0) - 1 && <Link data-testid="execution-header-next" href="#" onClick={next} className={classes.paginationLink}>&gt;&gt;&gt;</Link> }
+          { currentRunIndex >= (runCount || 0) - 1 && <span data-testid="execution-header-next" className={classes.paginationLink}>&gt;&gt;&gt;</span>}
+        </div>,
+        <table key="info-table">
+          <tbody>
+            <tr>
+              <td className={classes.configHeaderCell}>State</td>
+              <td data-testid="run-state" className={classes.configCell}>{!!currentRun && (currentRun.state.charAt(0).toUpperCase() + currentRun.state.slice(1))}</td>
+              <td className={classes.paddingCell}/>
+              <td className={classes.configHeaderCell}>Trigger</td>
+              <td data-testid="run-trigger" className={classes.configCell}>{!!currentRun && (currentRun.trigger.charAt(0).toUpperCase() + currentRun.trigger.slice(1))}</td>
+            </tr>
+            <tr>
+              <td className={classes.configHeaderCell}>Started at</td>
+              <td data-testid="run-started-at" className={classes.configCell}>{!!currentRun ? `${moment.utc(currentRun.started_at).format("MMMM Do YYYY, HH:mm:ss")} (UTC)` : ""}</td>
+              <td className={classes.paddingCell}/>
+              <td className={classes.configHeaderCell}>Completed at</td>
+              <td data-testid="run-completed-at" className={classes.configCell}>{!!currentRun?.completed_at ? `${moment.utc(currentRun.completed_at).format("MMMM Do YYYY, HH:mm:ss")} (UTC)` : ""}</td>
+            </tr>
+          </tbody>
+        </table>,
+        <Typography variant="body1" className={classes.outputComponentStyle} ref={outputRef} key="text-output">
+          {!!currentRun && <span data-testid="text-output">{currentRun.output}</span>}
+        </Typography>,
+        <div style={{ height: "4px" }} key="progress">
+          { !currentRun && <Progress /> }
+        </div>
+      ] }
+      { (runCount !== null && runCount === 0) && <Typography data-testid="no-run-msg" style={{ marginTop: "15px" }}>It looks like no export has run yet. Please come back later.</Typography>}
```

Comment:
Please change `marginTop` to `margin`. The text looks odd pushed so far to the left.

![image](https://github.com/OpsLevel/backstage-plugin/assets/83714714/bdee234c-3777-4eb4-88d8-d0920bd097d1)

Reply:
Also IMO we should point users to enabling auto-export:

```suggestion
      { (runCount !== null && runCount === 0) && <Typography data-testid="no-run-msg" style={{ margin: "15px" }}>Export has never run. Please ensure export is enabled in the configuration above.</Typography>}
```